day_two_guide/topics/proc_backing-up-project.adoc

```diff
@@ -13,12 +13,11 @@ Module included in the following assemblies:
 Creating a backup of all relevant data involves exporting all important
 information, then restoring into a new project.
 
-[NOTE]
+[IMPORTANT]
 ====
-Currently, a {product-title} project back up and restore tool is being developed
-by Red Hat. See the following bug for more information:
-
-* https://bugzilla.redhat.com/show_bug.cgi?id=1303205[bugzilla 1303205].
+The following procedure utilizes the `oc get all` command, which will not list
```

Comment:
I think you need to provide the steps for listing the PVCs and Secrets and backing them up. Please make it clearer that you just need the PVCs and Secrets, if that's true.

Also, in line 26, it says to list all the relevant data to back up, which isn't true anymore. Please update that line.